src/constants/index.ts

```diff
@@ -45,6 +45,13 @@ export const FRAX_3_POOL_NAME = "Frax 3Pool"
 export const TBTC_METAPOOL_V2_BTCV2_V3_NAME = "tBTCv2-BTCv2_v3"
 export const WCUSD_METAPOOL_V2_USDV2_V3_NAME = "wcUSD-USDv2_v3"
 export const SUSD_METAPOOL_V2_USDV2_V3_NAME = "sUSD-USDv2_v3"
+export const FRAX_USDC_POOL_NAME = "Saddle FRAX Pool"
```

Comment:
Could we use the same names that will be in the registry? eg https://github.com/saddle-finance/saddle-contract/pull/636/files#diff-2dd57de105b1d5d16969707bd002957c2a520fb6ec196fb6425f604de2085b81R20 "FRAX-USDC-BP", "FRAXBP-SUSD" etc